modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -684,13 +706,29 @@ private void _addObjectDefinitionResources(
 				).build();
 
 			for (ObjectRelationship objectRelationship : objectRelationships) {
+				String objectRelationshipName = objectRelationship.getName();
+
+				com.liferay.object.model.ObjectRelationship
+					oldObjectRelationship =
```

Comment:
@mateussgn rename to originalObjectRelationship to keep the modelListener pattern, [see](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/ObjectRelationshipModelListener.java#L48)